nasa-pictures-stealer/src/main/java/nasapicturesstealer/infrastructure/entity/NasaPictureEntity.java

```diff
@@ -0,0 +1,38 @@
+package nasapicturesstealer.infrastructure.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import org.springframework.data.annotation.CreatedDate;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "PICTURES")
+public class NasaPictureEntity {
+  
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @ToString.Exclude
+  @JoinColumn(name = "CAMERA_ID", nullable = false)
+  @ManyToOne(cascade = CascadeType.PERSIST)
+  private NasaCameraEntity nasaCameraEntity;
+  
+  private Integer nasaId;
+  
+  private String imgSrc;
+
+  @CreationTimestamp
+  private LocalDateTime createdAt;
```

Comment:
Use OffsetDateTime instead of LocalDateTime. With OffsetDateTime, you can easily convert to other time zones or to UTC for comparisons and calculations. It simplifies operations like finding the difference between times or aligning events in a global context. OffsetDateTime helps accurately represent the point in time regardless of the user's location.